compiler/noirc_evaluator/ssa_test/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "ssa_test"
```

Comment:
This sounds like it will test the SSA, which we do elsewhere. Maybe `ssa_verification` would be more suggestive of a broader scope?